jcl/src/java.management/share/classes/java/lang/management/ThreadInfo.java

```diff
@@ -39,14 +40,18 @@
  * @since 1.5
  */
 public class ThreadInfo {
+	
+	/* Set up access to ThreadInfo shared secret.*/
+	static {
+		ManagementAccessControl.setThreadInfoAccess(new ThreadInfoAccessImpl());
+	}
 
 	private long threadId;
 
 	/**
-	 * @brief Do /not/ rename this field; it is initialized but not used within the 
-	 * class's code (hence annotated "unused"), whereas, it is accessed via Reflection.
+	 * @brief Do /not/ rename this field; it is accessed via a package private 
```

Comment:
The `Do /not/ rename this field` part can be removed, as its now accesses via getNativeThreadId() instead of reflection.

Reply:
You could just remove the entire comment, as getNativeThreadId() is commented.